.github/workflows/publish-docker-image.yml

```diff
@@ -0,0 +1,39 @@
+name: Release multi-arch Docker Image
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Login to Docker Hub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Docker info
+      run: docker info
+    - name: Buildx inspect
+      run: docker buildx inspect
+
+    - name: Build and push image
+      uses: docker/build-push-action@v3
+      with:
+        context: .
+        file: ./Dockerfile
+        platforms: linux/amd64,linux/arm/v7,linux/arm64,linux/ppc64le,linux/s390x
+        # Only push if we are publishing a release
+        push: true
+        tags: testcontainers/ryuk:${{ github.event.release.tag_name }}
```

Comment:
Is this tag correct? I think it will override Ryuk, right?

Reply:
GAH! Yeah, this is a problem @eddumelendez DO NOT TAG vnc-recorder yet. I'll submit another PR with the fix.

Reply:
I already raised a PR.

Reply:
Resolved in #6.